pkgs/os-specific/linux/alsa-project/alsa-lib/default.nix

```diff
@@ -31,6 +32,8 @@ stdenv.mkDerivation rec {
 
   outputs = [ "out" "dev" ];
 
+  passthru.tests.pkg-config = testers.testMetaPkgConfig finalAttrs.finalPackage;
```

Comment:
Since this is always the same, wouldn't it be simpler (but potentially world rebuilding) to add it to `stdenv.mkDerivation` ?

Reply:
I rather repeat this boilerplate than make mkDerivation more antimodular. The former will get fixed while the latter will cause problems without enough people noticing or caring.

Reply:
https://github.com/NixOS/nixpkgs/issues/215162 tracking issue for this nuisance.